gnomad_qc/v5/resources/basics.py

```diff
@@ -252,6 +257,52 @@ def _aou_root_path() -> str:
 
 
 aou_bad_quality_path = f"{_aou_root_path()}/known_issues/wgs_v8_known_issue_1.txt"
+aou_genomic_metrics_path = (
+    f"{_aou_root_path()}/wgs/short_read/snpindel/aux/qc/genomic_metrics.tsv"
+)
+
+
+def get_invalid_aou_samples(path: str) -> hl.Table:
+    """
+    Import AoU genomic metrics.
+
+    :param path: Path to the genomic metrics file.
+    :return: Hail Table containing the genomic metrics.
+    """
+    types = {
+        "research_id": hl.tstr,
+        "sample_source": hl.tstr,
+        "site_id": hl.tstr,
+        "sex_at_birth": hl.tstr,
+        "dragen_sex_ploid" "y": hl.tstr,
+        "mean_coverage": hl.tfloat,
+        "genome_coverage": hl.tfloat,
+        "aou_hdr_coverage": hl.tfloat,
+        "dragen_contamination": hl.tfloat,
+        "aligned_q30_bases": hl.tint64,
+        "verify_bam_id2_contamination": hl.tfloat,
+        "biosample_collection_date": hl.tstr,
+    }
+    ht = hl.import_table(path, types=types).key_by("research_id")
```

Comment:
```suggestion
    ht = hl.import_table(aou_genomic_metrics_path, types=types).key_by("research_id")
```

Reply:
i think you can just directly pass in aou_genomic_metrics_path since i don't think we'll use this function for any other path